### DISCUSSION TITLE
Fleet screen improvements, multiple sorting / filtering

### DISCUSSION THREAD

**Discussion by morgun** [voxeltycoon/issues#413] [UI&UX]
It would be very useful to have an option to sort Fleet by second column by control clicking it (add asc, desc, remove column).
Also current sort selection is not visible right now and should have a darker or accent color.
![image](https://user-images.githubusercontent.com/1491443/109813760-6d5fb880-7c36-11eb-9d3a-b73200c8336a.png)

Example use case: You need to upgrade vehicles of certain type that transport a certain good.

Alternatively you can add filter option for each column.

**Comment by morgun** [comment]
Having a multiselect and search (filter elements by search string) in a filter would be also a great QoL addition.

**Comment by pommes** [comment]
As an additional idea for filtering:
When looking on the world screen one often wants to manage exactly the fleets visible on that current screen.
Train Fever for example has a filter mode that the fleet management window only shows fleets that are currently on the visible world screen.
